run_tests

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-
+set -x
 this_dir="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
 
 PYBOSSA_SETTINGS="${this_dir}/settings_test.py" \
     PYBOSSA_REDIS_CACHE_DISABLED=1 \
-    nosetests ${@:-test/}
+    nosetests -s ${@:-test/}
```

Comment:
can we remove `-s`? We can't see the whole test output in travis anymore

Reply:
Sure we can remove that.